merge_requests.go

```diff
@@ -113,11 +113,12 @@ type MergeRequest struct {
 	} `json:"changes"`
 	TimeStats *TimeStats `json:"time_stats"`
 	Squash    bool       `json:"squash"`
-	Pipeline  struct {
+	Pipeline  *struct {
```

Comment:
Even through this is perfectly valid, I don't really like having a pointer to an inline struct. So if you want to make this a pointer, then maybe we should declare a `MergeRequestPipeline` struct or reuse the existing `Pipeline` struct?

Reply:
@jippi any thoughts about this one?